src/panel/direction/road_map_panel.js

```diff
@@ -25,16 +26,22 @@ export default class RoadMapPanel {
   }
 
   preview() {
+    this.directionPanel.activePanel = this
```

Comment:
same thing, maybe we can call a callback to signal to the DirectionPanel instance that it is active. This way we don't need to directly handle the instance